mlir/test/Examples/standalone/test.toy

```diff
@@ -1,3 +1,5 @@
+# This doesn't substitute correctly right now
+# XFAIL: *
```

Comment:
We don't care about this directory. However, do you see the need of filing a ticket against ourselves in fixing the cmake version?

Reply:
This test used to pass. What caused it to fail?

Reply:
@krzysz00 I remember you mentioned that it is related with cmake. What's the last passing cmake version?

Reply:
I can try and take a look at it

Reply:
Need to get to the bottom of this.